src/Evolution/DiscontinuousGalerkin/ObserveFields.hpp

```diff
@@ -206,11 +215,17 @@ class ObserveFields<VolumeDim, tmpl::list<Tensors...>,
     EXPAND_PACK_LEFT_TO_RIGHT(record_tensor_components(
         tmpl::type_<NonSolutionTensors>{}, non_solution_tensors));
 
+    make_overloader(
+        [
+          this, &inertial_coordinates, &time, &components, &element_name
+        ](std::true_type /*has_analytic_solution*/, const auto& local_cache,
+          const db::item_type<
+              AnalyticSolutionTensors>&... local_analytic_solution_tensors)
+        noexcept {
     const auto analytic_solution =
```

Comment:
Check the formatting in this section.

Reply:
I didn't format as I wanted it to be easier to see what changed (and because clang-format left lines over 80 characters); but now that it is a simpler change I will format it properly...